src/ui/ViewModelBase.hh

```diff
@@ -0,0 +1,101 @@
+#ifndef RA_UI_VIEW_MODEL_BASE_H
+#define RA_UI_VIEW_MODEL_BASE_H
+#pragma once
+
+#include "ModelProperty.hh"
+
+#include <string>
+
+namespace ra {
+namespace ui {
+
+class ViewModelBase
```

Comment:
`ViewModelBase` serves as a `ModelProperty` bucket. It has no properties of its own.